src/actions/guides/database/intro_to_avram_and_orms.cr

```diff
@@ -125,15 +125,16 @@ class Guides::Database::IntroToAvramAndORMs < GuideAction
 
     # Configure your actual settings here
     MyOtherAdapter.configure do |settings|
+      # Refer to that adapter's configuration guide
       settings.url = "my_other_adapter_url"
     end
     ```
 
     * Place migrations (if necessary) in `db/migrations/`.
     * Boxes, and Queries are specific to Avram Models, but you can still use [Basic Operations](#{Guides::Database::ValidatingSaving.path}).
 
-    > If your app doesn't need a database, you should still set the `AppDatabase` configure setting to
-    > some non-empty string. Avram Operations can still be quite useful for things like contact forms,
+    > If your app doesn't need a database, you should still set the `AppDatabase` credentials setting to
+    > `Avram::Credentials.void`. Avram Operations can still be quite useful for things like contact forms,
```

Comment:
Are these lines needed since we have explicit docs above?

Reply:
just a little extra read for people skimming the docs and not actually reading, I guess.